me.njean.shootingpactris.json

```diff
@@ -0,0 +1,93 @@
+{
+    "app-id": "me.njean.shootingpactris",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "shpactris",
+    "finish-args": [
+                        "--socket=x11",
+                        "--share=network",
+                        "--device=all",
+                        "--socket=pulseaudio"
+                    ],
+    "modules": [
+        "shared-modules/glu/glu-9.json",
+        {
+            "name": "scons",
+            "buildsystem": "simple",
+            "cleanup": ["*"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "7801f3f62f654528e272df780be10c0e9337e897650b62ddcee9f39fde13f8fb",
+                    "url": "https://downloads.sourceforge.net/project/scons/scons/3.1.2/scons-3.1.2.tar.gz"
+                }
+            ],
+            "build-commands": [ "python3 setup.py install --prefix=/app" ]
+        },
+        {
+            "name": "godot",
+            "buildsystem": "simple",
+            "cleanup": ["*"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "556b564079fb1b21a05320c359268b2032c87d8b3bfaa4fd104ebf9c909ab87d",
+                    "url": "https://downloads.tuxfamily.org/godotengine/3.2.1/godot-3.2.1-stable.tar.xz"
+                }
+            ],
+            "build-commands": [
+                "python3 /app/bin/scons -j$FLATPAK_BUILDER_N_JOBS platform=server tools=yes target=release_debug",
+                "install -Dm755 bin/godot_server.x11.opt.tools.* /app/bin/godot-server"
+            ]
+        },
+        {
+            "name": "godot-templates",
+            "buildsystem": "simple",
+            "cleanup": ["*"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "c6883b2dd8848b4ea90d869b971a98364296482d2223ae20e61cb492fe082abb",
+                    "url": "https://downloads.tuxfamily.org/godotengine/3.2.1/Godot_v3.2.1-stable_export_templates.tpz",
+                    "dest-filename": "templates.zip"
+                }
+            ],
+            "build-commands": [
+                "install -Dm755 -t /app/share/godot/templates/3.2.1.stable/ linux_x11_*"
+                ]
+        },
+        {
+            "name": "shpactris",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://framagit.org/nicooo/shpactris",
+                    "tag": "v1.0-online-beta2",
+                    "commit": "84d5e4c3976fbdd0ebe4eb55f7ec7bff8aba4fa1"
+                },
+                {
+                    "type": "file",
+                    "path": "me.njean.shootingpactris.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "me.njean.shootingpactris.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Fixed.